llvm/tools/repo-fragments/RepoFragments.cpp

```diff
@@ -0,0 +1,124 @@
+//===- repo-fragments - Dump the names and digests using a ticket file. --===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/MC/MCRepoTicketFile.h"
+#include "llvm/Support/CommandLine.h"
+
+#include "pstore/dump/mcrepo_value.hpp"
+
+using namespace llvm;
+
+namespace {
+
+static cl::OptionCategory ToolOptions("Tool Options");
```

Comment:
You don’t need to say `static` when the variable is defined inside an anonymous namespace.

Reply:
Thanks, I have removed `static` for all command option variables.